WooCommerce/src/main/kotlin/com/woocommerce/android/ui/main/MainActivity.kt

```diff
@@ -407,15 +408,11 @@ class MainActivity :
     }
 
     private fun showToolbar() {
-        if (restoreToolbarHeight > 0) {
-            binding.collapsingToolbar.layoutParams.height = restoreToolbarHeight
-            restoreToolbarHeight = 0
-        }
+        binding.collapsingToolbar.show()
     }
 
     fun hideToolbar() {
```

Comment:
Np this can be private now.

```suggestion
    private fun hideToolbar() {
```